tests/pipeline/test_course_daily_metrics.py

```diff
@@ -202,7 +209,9 @@ def test_get_days_to_complete(self):
                         errors=[])
         actual = pipeline_cdm.get_days_to_complete(
             course_id=self.course_overview.id,
-            date_for=self.today)
+            date_for=self.today + datetime.timedelta(
+                days=1 + max(self.cert_days_to_complete))
+        )
```

Comment:
The date filter in this method checks for timestamp so even if the date is the same and but time is different the filter misses. This is to cater that